src/umd/ReactUMDEntry.js

```diff
@@ -17,17 +17,8 @@ var React = require('React');
 var ReactUMDEntry = Object.assign({
   __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED: {
     ReactCurrentOwner: require('react/lib/ReactCurrentOwner'),
+    ReactComponentTreeHook: require('react/lib/ReactComponentTreeHook'),
```

Comment:
This part is the piece I'm skeptical about.

We only needed this export because the debug ids are shared. We didn't need to export this in prod before Fiber.

Stacks through `getStackAddendumByWorkInProgressFiber` doesn't use the debug ids. I.e. it doesn't rely on global mutable state and therefore can be completely decoupled.

We only need prod stacks in Fiber through `getStackAddendumByWorkInProgressFiber` if I understand it correctly.

Therefore, why doesn't we just put `getStackAddendumByWorkInProgressFiber` in the renderer so we don't need this connection between the packages in prod?